server/tserver/src/main/java/org/apache/accumulo/tserver/log/RecoveryLogsIterator.java

```diff
@@ -61,6 +65,9 @@
    */
   public RecoveryLogsIterator(ServerContext context, List<Path> recoveryLogDirs, LogFileKey start,
       LogFileKey end, boolean checkFirstKey) throws IOException {
+    var conf = context.getConfiguration();
+    var indexCacheSize = getMemoryAsBytes(conf.get(Property.TSERV_INDEXCACHE_SIZE));
```

Comment:
May want dedicated props for the recovery cache size.  For example I may want a 10G cache for scans and 256M cache for recovery.